### PR TITLE
Add Selenium test for openapi-3.1

### LINKED ISSUE #25078: Add Selenium tests for the openapi-3.1 UI
Similar to #20512 but for the UI provided by the `openapi-3.1` feature.

Test both endpoints:

- `/api/explorer`
- `/ibm/api/explorer` (requires authentication)

### PR DESCRIPTION
**Pull request by Azquelt**
Add a selenium test for the `openapi-3.1` feature endpoints.

Additionally:
- move some utility methods into fattest.simplicity so they can be reused across different test projects
- Update the `mpOpenAPI-3.1` UI test to use `Color.fromString` [as recommended in the Selenium docs](https://www.selenium.dev/documentation/webdriver/support_features/colors/)

Fixes #25078

### CODE REVIEW

**Comment by Azquelt** [comment]
#build
#libby

File: dev/com.ibm.ws.openapi.ui_fat/fat/src/com/ibm/ws/openapi/ui/fat/tests/UIBasicTest.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * A basic test that we can open the UI and that it appears to have loaded correctly
+ */
+@RunWith(FATRunner.class)
+public class UIBasicTest {
+
+    /** Wait for "long" tasks like initial page load or making a test request to the server */
+    private static final Duration LONG_WAIT = Duration.ofSeconds(30);
```
Comment (tevans78) [inline dev/com.ibm.ws.openapi.ui_fat/fat/src/com/ibm/ws/openapi/ui/fat/tests/UIBasicTest.java:55]: I realise that this is not yet used anywhere else, but in the long run might it be? In which case would it be better as a public constant in the SeleniumWaits class?

Comment (Azquelt) [reply dev/com.ibm.ws.openapi.ui_fat/fat/src/com/ibm/ws/openapi/ui/fat/tests/UIBasicTest.java:55]: Possibly, but I'm not that confident about whether this is the correct value.

**Review by tevans78 — APPROVED** [review]
LGTM

**Comment by Azquelt** [comment]
Failures in the build do not appear to be related to this change.

**Pull request merged**